frontend/lib/user/repository.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+import boto3
+
+
+class UserDynamoDBRepository:
+
+    def __init__(self, endpoint_url, region_name, table_name):
+        self.endpoint_url = endpoint_url
+        self.region_name = region_name
+        self.table_name = table_name
+
+    def get_table(self):
+        dynamodb = boto3.resource(
+            'dynamodb',
+            endpoint_url=self.endpoint_url,
+            region_name=self.region_name
+        )
+        table = dynamodb.Table(self.table_name)
+        return table
+
+    def delete(self, username, table=None):
+        if table is None:
+            table = self.get_table()
+
+        return table.delete_item(Key={'username': username})
+
+    def put(self, username, manage_users, queues, table=None):
+        if table is None:
+            table = self.get_table()
+
+        return table.put_item(Item={
+            'username': username,
+            'permissions': {
+                'manage_users': manage_users,
+                'queues': queues
+            }
+        })
+
+    def get(self, username, table=None):
+        if table is None:
+            table = self.get_table()
+        result = table.get_item(Key={'username': username})
+        return result.get('Item')
+
+    def scan(self, table):
```

Comment:
let's call it list (scan is a dynmodb term) we want to get a list of all users (which internally will use a scan)